### ISSUE TITLE
Error during installation 

### ISSUE THREAD

**Issue by HumanG33k** [iv-org/documentation#80]
I follow instruction for debian. 

``` bash
shards update; shards install
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/iv-org/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/luislavena/radix.git
Unable to satisfy the following requirements:

- `crystal (>= 0.35.0, < 2.0.0)` required by `pg 0.23.2`
- `crystal (>= 0.35.0, < 2.0.0)` required by `sqlite3 0.18.0`
- `crystal (~> 0.35, >= 0.35.0)` required by `kemal 0.27.0`
- `crystal (*)` required by `pool 0.2.4`
- `crystal (>= 0.34.0, < 2.0.0)` required by `protodec 0.1.4`
- `crystal (~> 0.35, >= 0.35.1)` required by `lsquic 2.18.1-1`
- `crystal (>= 0.35.0, < 2.0.0)` required by `db 0.10.1`
- `crystal (< 1.0.0)` required by `radix 0.3.9`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/luislavena/radix.git
Fetching https://github.com/jeromegn/kilt.git
Fetching https://github.com/crystal-loot/exception_page.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/iv-org/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Unable to satisfy the following requirements:

- `crystal (>= 0.35.0, < 2.0.0)` required by `pg 0.23.1`
- `crystal (>= 0.35.0, < 2.0.0)` required by `sqlite3 0.18.0`
- `crystal (~> 0.35, >= 0.35.0)` required by `kemal 0.27.0`
- `crystal (< 1.0.0)` required by `pool 0.2.3`
- `crystal (~> 0.34, >= 0.34.0)` required by `protodec 0.1.3`
- `crystal (~> 0.35, >= 0.35.1)` required by `lsquic 2.18.1-1`
- `crystal (~> 0.35, >= 0.35.0)` required by `db 0.10.0`
- `crystal (< 1.0.0)` required by `radix 0.3.9`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.
```

**Comment by hackerncoder** [comment]
You need to install an older version of crystal right now. See https://github.com/iv-org/invidious/issues/1933 for status on updating to crystal 1.0.

**Comment by SamantazFox** [comment]
Ah, oops, yeah, I forgot to update debian instructions last time :x

**Comment by SamantazFox** [comment]
@HumanG33k now invidious supports Crystal v1.0.0! You can try again :)

**Comment by HumanG33k** [comment]
Ok probably need to add a section in readme.

In order to not have 
```
Unable to satisfy the following requirements:

- `crystal (>= 0.35.0, < 2.0.0)` required by `pg 0.23.2`
- `crystal (>= 0.35.0, < 2.0.0)` required by `sqlite3 0.18.0`
- `crystal (~> 0.35, >= 0.35.0)` required by `kemal 0.27.0`
- `crystal (*)` required by `pool 0.2.4`
- `crystal (>= 0.34.0, < 2.0.0)` required by `protodec 0.1.4`
- `crystal (~> 0.35, >= 0.35.1)` required by `lsquic 2.18.1-1`
- `crystal (>= 0.35.0, < 2.0.0)` required by `db 0.10.1`
- `crystal (< 1.0.0)` required by `radix 0.3.9`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.
```
i did a 

```
rm shard.lock
```

I will edit after command end.

**Comment by SamantazFox** [comment]
Fixed in #84 and #85
